### DISCUSSION TITLE
Add Node Graph Data Link

### DISCUSSION THREAD

**Discussion by JohnBolander** [grafana/grafana#65726] [Dashboards, Panels & Visualizations]
The [docs for the node graph](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/node-graph/) say "There also can be additional links in the context menu that can target either other parts of Grafana or any external link." There does not seem to be a way to add a data link to the column so that it views in the node (see first image from the [latest grafana](https://play.grafana.org/d/Pbk2rfBGz/node-graph?orgId=1&editPanel=2). The data source in gdev-testdata source seems to add the link some other way. Can the ability to add a data link to the node graph be added so that it works for other general datasources the say way you see the link in the example?

<img width="916" alt="image" src="https://user-images.githubusercontent.com/4633707/229171473-a73d3f63-8c2e-4495-8904-0cfaf7386bc8.png">
<img width="654" alt="image" src="https://user-images.githubusercontent.com/4633707/229171598-5c5f43e9-e135-4115-b1f4-af14069fdf36.png">

**Comment by zzzzl13** [comment]
Also wonder how to add the link

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82739. Feel free to continue the discussion around this there. Thank you!
